src/UInt.h

```diff
@@ -0,0 +1,53 @@
+// Copyright © 2017-2019 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#pragma once
+
+#include "Data.h"
+
+#include <boost/multiprecision/cpp_int.hpp>
+#include <boost/multiprecision/cpp_int/cpp_int_config.hpp>
+
+namespace TW {
+
+namespace bm = boost::multiprecision;
+
+using int160_t = bm::number<bm::backends::cpp_int_backend<160, 160, bm::signed_magnitude, bm::unchecked, void>>;
+using uint160_t = bm::number<bm::backends::cpp_int_backend<160, 160, bm::unsigned_magnitude, bm::unchecked, void>>;
+
+template<class T>
+inline T load(const Data &data, int initial_position = 0) {
```

Comment:
can we merge this with `uint256.h` together somehow?

Reply:
done